tsconfig.json

```diff
@@ -27,7 +27,9 @@
       "dom"
     ],
     // Skip type checking of all declaration files (*.d.ts).
-    "skipLibCheck": true
+    "skipLibCheck": true,
+
+    "module": "ESNext"
```

Comment:
this is worth noting, due to using dynamic import in https://github.com/adobe-private/react-spectrum-v3/pull/488/files#diff-eda1b2489d37703427876c722cfe813bR45